src/common/i18n/translations/type.ts

```diff
@@ -1,4 +1,8 @@
 export type Translations = {
+  blockUser: {
+    header: string;
+    body: string;
+  };
```

Comment:
Added the blockUser translation so that the display for the text could be done in both Chinese and English.